t/ui/18-tests-details.t

```diff
@@ -81,7 +81,10 @@ sub find_candidate_needles {
     # save implicit waiting time as long as we are only looking for elements
     # that should be visible already
     disable_timeout;
-    return {} unless $candidates_menu->is_enabled;
+    unless ($candidates_menu->is_enabled) {
+        enable_timeout;
+        return {};
+    }
```

Comment:
Good catch.

I don't think disabling the timeout has any effect on `is_enabled` here. So we can probably just move the early return before `disable_timeout`.

Reply:
Yes, I could reproduce it before, usually after less than 20 runs.

Then I added the `enable_timeout` thing before the retun, and from then on it started to fail even more often, inside of the `wait_for_element`, which is when I started looking for other places where we call a module step and saw that we always do a `wait_for_ajax` there.

Reply:
I changed it as you suggested and moved the `disable_timeout` below